### ISSUE TITLE
False positive: Use a pipeline when there are nested function calls

### ISSUE THREAD

**Issue by general-CbIC** [rrrene/credo#1040]
### Environment

```
❯ mix credo -v
1.7.0

❯ elixir -v

Erlang/OTP 25 [erts-13.1] [source] [64-bit] [smp:16:16] [ds:16:16:10] [async-threads:1] [jit:ns]

Elixir 1.14.3 (compiled with Erlang/OTP 25)

❯ uname -som

Darwin x86_64
```

### What were you trying to do?

Created this module:

```elixir
defmodule CredoIssue do
  @moduledoc false
  def check(some_value) do
    "Inspect: #{inspect(some_value)}"
  end
end
```

Allowed check named `NestedFunctionCalls` in `.credo.exs`.

### Expected outcome

No warnings, since there is only one call `inspect(some_value)` inside the string interpolation.

### Actual outcome

```
  Code Readability
┃
┃ [R] → Use a pipeline when there are nested function calls
┃       lib/credo_issue.ex:4 #(CredoIssue.check)
```

**Comment by randycoulman** [comment]
I think this is the same issue as #1034.

**Comment by general-CbIC** [comment]
Yes, thank you 🙏🏻
